src/app/pages/home-page/offer-item-page/offer-item-page.component.html

```diff
@@ -43,12 +42,13 @@
         </div>
       </div>
     </mat-tab>
-    <mat-tab label="addresses">
-      <a
-        *ngFor="let office of offer.offices"
-        class="address link-reset"
-        [routerLink]="['/home/offices', office.id]"
-      >
+
+    <mat-tab label="{{ 'CARD.ADDRESSES' | translate }}">
+      <div *ngFor="let office of offices" class="address">
+        <a
+          class="address__link-overlay link"
+          [routerLink]="['/home/offices', office.id]"
+        ></a>
         <mat-icon class="address__location-icon">location_on</mat-icon>
```

Comment:
icon styles better get from shared icon styles file

Reply:
there in styles only margin. I don't know, but maybe it's just a local style?

```
  &__location-icon {
    margin-right: 0.5em;

    @media screen and (max-width: $mobile) {
      margin-right: 0.3em;
    }
  }
  ```